server/webhook.go

```diff
@@ -205,3 +213,192 @@ func (p *Plugin) getSubscriptionsWebhookURL(instanceID types.ID) string {
 	v.Add("secret", cf.Secret)
 	return p.GetPluginURL() + instancePath(routeAPISubscribeWebhook, instanceID) + "?" + v.Encode()
 }
+
+func (wh *webhook) checkIssueWatchers(p *Plugin, instanceID types.ID) {
+	if !wh.eventTypes.ContainsAny("event_created_comment") {
+		return
+	}
+
+	jwhook := wh.JiraWebhook
+	commentAuthor := mdUser(&jwhook.Comment.UpdateAuthor)
+	commentMessage := fmt.Sprintf("%s **commented** on %s:\n>%s", commentAuthor, jwhook.mdKeySummaryLink(), jwhook.Comment.Body)
+	client, connection, err := wh.fetchConnectedUser(p, instanceID)
+	if err != nil || client == nil {
+		return
+	}
+
+	watcherUsers, err := client.GetWatchers(instanceID.String(), wh.Issue.ID, connection)
+	if err != nil {
+		p.errorf("error while getting watchers for issue , err : %v", err)
+		return
+	}
+
+	for _, watcherUser := range watcherUsers.Watchers {
+		if wh.checkNotificationAlreadyExist(watcherUser.DisplayName, watcherUser.AccountID) {
+			return
+		}
```

Comment:
current struct is like this  for watches 
instead of name it is displayname 

```
type Watches struct {
	Self       string     `json:"self,omitempty" structs:"self,omitempty"`
	WatchCount int        `json:"watchCount,omitempty" structs:"watchCount,omitempty"`
	IsWatching bool       `json:"isWatching,omitempty" structs:"isWatching,omitempty"`
	Watchers   []*Watcher `json:"watchers,omitempty" structs:"watchers,omitempty"`
}

// Watcher represents a simplified user that "observes" the issue
type Watcher struct {
	Self        string `json:"self,omitempty" structs:"self,omitempty"`
	Name        string `json:"name,omitempty" structs:"name,omitempty"`
	AccountID   string `json:"accountId,omitempty" structs:"accountId,omitempty"`
	DisplayName string `json:"displayName,omitempty" structs:"displayName,omitempty"`
	Active      bool   `json:"active,omitempty" structs:"active,omitempty"`
}
```

Reply:
FYI `watcherUser.Name` will likely be used for Jira Server, as `AccountID` is not available in Jira Server